pyramid_openapi3/__init__.py

```diff
@@ -182,6 +216,16 @@ def explorer_view(request: Request) -> Response:
             route_name=route_name, permission=permission, view=explorer_view
         )
 
+        def redirect_view(request: Request) -> FileResponse:
+            return FileResponse(redirect_html.abspath())
+
+        config.add_route(oauth_redirect_route_name, oauth_redirect_route)
```

Comment:
Could/Should the oauth view/route be registered only if needed, i.e. if one of the config settings is present, otherwise skipped by default?

Reply:
My original impl had the feature as opt in via an enable_oauth_redirect setting but as I kept working on it I came to the conclusion that it’s just part of the swagger ui and would cause no harm to anyone as it’s just a small html page and it simplified things to just leave it enabled.

Reply:
If you want me to bring back the setting I’m willing but that was my progression.